rust/src/controllers/emergency.rs

```diff
@@ -88,21 +89,27 @@ pub fn update_state(
 
 fn send_emergency_notifications(
     conn: &mut PostgresConnection,
-    channel: &Channel,
     username: &String,
     state: &UserState,
 ) -> Result<(), APIInternalError> {
     let sender_details = get_user_details(username, conn)
         .map_err(APIInternalError::from_db_err)?
         .unwrap();
 
-    get_emergency_connections(conn, username)
+    let values = get_emergency_connections(conn, username)
         .map_err(APIInternalError::from_db_err)
-        .map(|recipients| build_recipients_notifications(conn, recipients, &sender_details, state))
-        .and_then(|values| {
-            send_notification(channel, json!(values).to_string())
-                .map_err(APIInternalError::from_db_err)
-        })
+        .map(|recipients| {
+            build_recipients_notifications(conn, recipients, &sender_details, state)
+        })?;
+
+    let mut rabbit_conn = RabbitConnection::insecure_open(&get_rabbitmq_uri())
```

Comment:
So this is ok for now, but I think it would make the server crash if it fails to open rabbitmq?